### PR TITLE
Microsemi rebased

### PR DESCRIPTION
**Pull request by zaddan**
changes made to integrate microsemi platform

### CODE REVIEW

File: src/main/scala/ip/xilinx/Xilinx.scala [hunk 1]
```diff
@@ -241,38 +241,14 @@ class vc707_sys_clock_mmcm2 extends BlackBox {
...
-    CONFIG.CLKOUT3_PHASE_ERROR {105.461} \
-    CONFIG.CLKOUT4_JITTER {157.199} \
-    CONFIG.CLKOUT4_PHASE_ERROR {105.461} \
-    CONFIG.CLKOUT5_JITTER {136.686} \
-    CONFIG.CLKOUT5_PHASE_ERROR {105.461} \
-    CONFIG.CLKOUT6_JITTER {126.399} \
-    CONFIG.CLKOUT6_PHASE_ERROR {105.461} \
-    CONFIG.CLKOUT7_JITTER {206.010} \
-    CONFIG.CLKOUT7_PHASE_ERROR {136.686}] [get_ips vc707_sys_clock_mmcm2] """
+    ] [get_ips vc707_sys_clock_mmcm2] """
```
Comment (terpstra) [inline src/main/scala/ip/xilinx/Xilinx.scala]: You almost certainly did not want to commit this.

**Comment by terpstra** [comment]
Make sure you use 'Create a Merge Commit' and not 'Squash and merge'

**Comment by yunsup** [comment]
Can we delete comments in the code, please?

**Comment by terpstra** [comment]
I assume you mean the commented-out code, not the comments in the code.

Certainly commented-out scala code can be removed, as well as the
superfluous tool-generated comments in the pdc file. However, I think we
want to keep the IO assignments that are commented out. We will likely want
to re-activate that code once we have more functional shells.

**Pull request merged**